### PR TITLE
Add CLP/UGX exponent change

### PR DESCRIPTION
**Pull request by ct-clearhaus**

### CODE REVIEW

File: source/index.md [hunk 1]
```diff
@@ -805,6 +805,17 @@ https://gateway.clearhaus.com/cards/:id
 https://gateway.clearhaus.com/account
 ````
 
+## Changes
+
+### CLP and UGX exponent change
+
+CLP and UGX changes from exponent 2 to exponent 2.
```
Comment (mt-clearhaus) [inline source/index.md]: s/changes/change/
s/to exponent 2/to exponent 0/

File: source/index.md [hunk 2]
```diff
@@ -805,6 +805,17 @@ https://gateway.clearhaus.com/cards/:id
...
+
+Between 2017-10-12T19:00:00+00:00 and 2017-10-15T19:00:00+00:00 (both inclusive)
+it will not be possible to process transactions in neither CLP nor UGX; before
```
Comment (mt-clearhaus) [inline source/index.md]: Is this double negation? ("not" and "neither") How about "not be possible to ... in CLP and UGX"?

File: source/index.md [hunk 3]
```diff
@@ -805,6 +805,17 @@ https://gateway.clearhaus.com/cards/:id
...
+this timespan, the exponent is 2, after the timespan, the exponent is 0.
```
Comment (mt-clearhaus) [inline source/index.md]: Maybe s/2, after/2; after/

Comment (ct-clearhaus) [inline source/index.md]: I'm not fan of the wording; improvements are more than welcome 🙂

File: source/index.md (hunk 2 above)
Comment (ct-clearhaus) [reply source/index.md]: I tried an even more direct wording. WDYT?

File: source/index.md (hunk 3 above)
Comment (mt-clearhaus) [reply source/index.md]: Maybe have "before this ..." in its own sentence, i.e. period after "UGX"?

File: source/index.md [hunk 4]
```diff
@@ -807,13 +807,13 @@ https://gateway.clearhaus.com/account
 
 ## Changes
 
-### CLP and UGX exponent change
+### CLP and UGX exponent changes
 
-CLP and UGX changes from exponent 2 to exponent 2.
+CLP and UGX changes from exponent 2 to exponent 0.
```
Comment (mt-clearhaus) [inline source/index.md]: I was actually referring to "changes" after "CLP and UGX" :wink: However, I like the change in the section title :slightly_smiling_face:

Comment (ct-clearhaus) [reply source/index.md]: I'm lost. Which line? What diff?

**Pull request merged**